app/services/quote_service.rb

```diff
@@ -0,0 +1,26 @@
+class QuoteService
+  
+  def self.conn
+    Faraday.new(url: 'api.quotable.io/') #do |faraday|
```

Comment:
I think the `url` here might be causing the problem with your API call. Try updating to: `'https://api.quotable.io/'`

With this update, the `get_search_quotes_api_call` method works for me and returns a response.